.golangci.yml

```diff
@@ -155,6 +156,7 @@ linters:
     - maintidx
     - nestif
     - nilnil # we return "nil, nil"
+    - nolintlint
```

Comment:
just curious, what's the issue there?

Reply:
It shows a lot of false positives for some reason.